ceilometer/manifests/client/compute.pp

```diff
@@ -0,0 +1,42 @@
+#
+# Installs the ceilometer python library.
+#
+# == parameters
+#  [*ensure*]
+#    ensure state for pachage.
+#
+class ceilometer::client::compute (
+  $ensure = 'present'
+) {
+
+  include ::ceilometer::params
+
+  package { 'python-ceilometerclient':
```

Comment:
Thanks for pointing that out. When I was first puppetizing the ceilometer agents, I realized that the configurations for the compute agents are different than the controller agents. Because of this, I decided to create a new client/compute.pp manifest and not modify client.pp. However, I'll try to refactor this, so that the changes introduced to client/compute.pp are in agent/compute.pp instead.